decidim-initiatives/spec/events/decidim/initiatives/admin/support_threshold_reached_event_spec.rb

```diff
@@ -6,15 +6,12 @@
   include_context "when a simple event"
 
   let(:event_name) { "decidim.events.initiatives.support_threshold_reached" }
-  let(:resource) { initiative }
-
-  let(:initiative) { create :initiative }
-  let(:participatory_space) { initiative }
-  let(:initiative_title) { decidim_html_escape(translated(initiative.title)) }
+  let(:resource) { create :initiative, title: generate_localized_title(:initiative_title) }
```

Comment:
This is actually needed, otherwise, the pipeline will fail. 
![image](https://github.com/decidim/decidim/assets/105683/d10c611a-d920-419a-9923-96878626c6e6)